MainModule/Server/Commands/Moderators.lua

```diff
@@ -6260,20 +6260,24 @@ return function(Vargs, env)
 						return true
 					end
 				end
-
 				for i,v in pairs(service.GetPlayers(plr,args[1])) do
-					local isMuted = false
+					local _isMuted = false
+					local _isContributor = false
+					local _isBanned = false
+					if v.GetRankInGroup(886423) = 10 then
```

Comment:
`Player.GetRankInGroup` is called with `:` (self=Player) and you are calling the function without `:` and not providing the 1st argument as self (Player). You should do `v:GetRankInGroup(886423)` or `v.GetRankInGroup(v, 886423)`.

Reply:
Understood big brain